.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,14 @@
-<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
 ## Summary of the Pull Request
 
-<!-- Other than the issue solved, is this relevant to any other issues/existing PRs? --> 
-## References
+## References and Relevant Issues
```

Comment:
Isn't "Detailed Description of the Pull Request / Additional comments" redundant with "Summary of the Pull Request"? If not, should they maybe be next to each other?

Reply:
Meh, I always read it as more:
* An abstract, a quick summary
* the required checkboxes
* the more in depth discussion of technical aspects.